src/utils.ts

```diff
@@ -31,6 +31,17 @@ export const DEFAULT_GROUP = [
   "Utilities",
 ];
 
+export const DEFAULT_COLOR = [
```

Comment:
Since it's a fixed and unchanging choice for colors, maybe we can use enum for better types